Demisto-sdk release 1.36.2 (#4863)
demisto/demisto-sdk: CHANGELOG.md

# Changelog
## 1.36.2 (2025-03-16)
### Feature

> * Added support for playbook tags in metadata dump, and support for `supportedModules` field. [#4832](https://github.com/demisto/demisto-sdk/pull/4832)

Reviewer: ```suggestion
* Added support for playbook tags in metadata dump, and support for *supportedModules* field. [#4832](https://github.com/demisto/demisto-sdk/pull/4832)
```

### Fix

> * Resolved an issue in external contributions pull requests where incorrect Git status fetching led to validation failures. [#4829](https://github.com/demisto/demisto-sdk/pull/4829)

Reviewer: ```suggestion
* Fixed an issue in external contributions pull requests flow where incorrect Git status fetching led to validation failures. [#4829](https://github.com/demisto/demisto-sdk/pull/4829)
```

> * Fixed an issue where pre-commit failed on deleted content folders. [#4855](https://github.com/demisto/demisto-sdk/pull/4855)

Reviewer: ```suggestion
* Fixed an issue where ***pre-commit*** failed on deleted content folders. [#4855](https://github.com/demisto/demisto-sdk/pull/4855)
```

### Internal

> * fixed an issue with GitHub actions. [#4856](https://github.com/demisto/demisto-sdk/pull/4856)

Reviewer: ```suggestion
* Fixed an issue with GitHub actions. [#4856](https://github.com/demisto/demisto-sdk/pull/4856)
```

> * Fix the graph schemas to support supportsquickactions in the correct place. [#4853](https://github.com/demisto/demisto-sdk/pull/4853)

Reviewer: ```suggestion
* Updated the Graph schemas to support *supportsquickactions* in the correct place. [#4853](https://github.com/demisto/demisto-sdk/pull/4853)
```
